### PR TITLE
Makes Baseballs and Rabbits and small-sized and able to be picked up and held.

### LINKED ISSUE #77509: bunny doesnt fit into pet carrier
Reporting client version: 514.1589

## Round ID:
[212218](https://scrubby.melonmesa.com/round/212218)

## Testmerges:
- [Trying to qdelete a callback now lags the server instead.](https://www.github.com/tgstation/tgstation/pull/77447)
- [Fixes some lua problems, specifically with qdeling callbacks and adds lua utility functions](https://www.github.com/tgstation/tgstation/pull/77468)

### PR DESCRIPTION
**Pull request by FernandoJ8**
## About The Pull Request

What it says on the tin. They previously inherited the size of a human from `/obj/item/beach_ball` and `/mob/living` respectively.
Also allows rabbits to be picked up and held.

## Why It's Good For The Game

Fixes #77509.
Baseballs shouldn't be as large as a beachball.
Bunnies should be small. Proof:

## Changelog
:cl:
fix: rabbits are now small-sized rather than human-sized, as well as capable of being picked up
fix: baseballs are now small sized rather than large
/:cl:

### CODE REVIEW

**Review by san7890 — APPROVED** [review]

**Comment by AviiCinders** [comment]
Wrong
https://www.youtube.com/watch?v=VgvgY0igIW4
(Also make it so we can pick them up pls)

**Comment by FernandoJ8** [comment]
> Wrong https://www.youtube.com/watch?v=VgvgY0igIW4 (Also make it so we can pick them up pls)

So called "world's largest rabbit", yet a person can still easily hold them.
WEAK.
But yes, good point.

**Comment by AviiCinders** [comment]
> > Wrong https://www.youtube.com/watch?v=VgvgY0igIW4 (Also make it so we can pick them up pls)
> 
> So called "world's largest rabbit", yet a person can still easily hold them. WEAK. But yes, good point.

Give this man all the GBP

**Comment by FernandoJ8** [comment]
Noticed this as well and thought it was related enough to add it to the PR rather than making a new one.

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -917,6 +917,7 @@
 	icon = 'icons/obj/toys/balls.dmi'
 	icon_state = "baseball"
 	inhand_icon_state = "baseball"
+	w_class = WEIGHT_CLASS_SMALL
```
Comment (Ghommie) [inline code/game/objects/items/toys.dm:920]: Uuuh, how's this related to this PR? Though, it's a one line change, so I don't mind as long as you add it to the changelog.

**Pull request merged**